crates/consensus-logic/src/sync_manager.rs

```diff
@@ -83,6 +95,8 @@ pub fn start_sync_tasks<E: ExecEngineCtl + Sync + Send + 'static>(
     // Create channels.
     let (fcm_tx, fcm_rx) = mpsc::channel::<ForkChoiceMessage>(64);
     let (csm_tx, csm_rx) = mpsc::channel::<CsmMessage>(64);
+    let (chain_msg_tx, chain_msg_rx) = mpsc::channel::<ChainWorkerMessage>(64);
+    let (exec_tx, exec_rx) = strata_eectl::handle::make_handle_pair();
```

Comment:
See other comments.  I know this pattern exists literally in the same space but we can try to avoid perpetuating it.